src/core/jsonld.js

```diff
@@ -0,0 +1,91 @@
+// Module core/jsonld
+// Create script element with JSON-LD for SEO document description
+export const name = "core/jsonld";
+
+export function run(conf, doc, cb) {
+  if (!conf.doJsonLd) {
+    return cb();
+  }
+  // Content for JSON
+  const types = ["TechArticle"];
+  if (conf.rdfStatus) types.push(conf.rdfStatus);
+
+  const jsonld = {
+    "@context": ["http://schema.org", {
+      "@vocab": "http://schema.org/",
+      "@language": "en",
+      "w3p": "http://www.w3.org/2001/02pd/rec54#",
+      "foaf": "http://xmlns.com/foaf/0.1/",
+      "datePublished": {"@type": "xsd:date"},
+      "inLanguage": {"@language": null},
+      "isBasedOn": {"@type": "@id"},
+      "license": {"@type": "@id"}
+    }],
+    id: conf.canonicalURI || conf.thisVersion,
+    type: types,
+    name: conf.title,
+    inLanguage: doc.documentElement.getAttribute("lang") || "en",
+    license: conf.licenseInfo.url,
+    datePublished: conf.dashDate,
+    copyrightHolder: {
```

Comment:
I thought about that, as well as `overrideCopyright`, but we don't get quite the information out of this. But, presuming that it's a simple text string, could create something like the following:

```json
  copyrightHolder: [{
      name: "World Wide Web Consortium",
      url: "https://www.w3.org/"
    }, {
      name: ${conf.additionalCopyrightHolders}
    }
```

although, need to consider that it may be an array.

Similarly, the w3c bit could be left out if there is an `overrideCopyright`. There's also the non-w3c case, but I wasn't clear on what could be gleaned from the configuration in that case.

Reply:
The problem with `overrideCopyright` is that it's described as being markup, which isn't too helpful here.